src/components/dls/Toggle/Toggle.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+
+import Toggle from './Toggle';
+
+export default {
+  title: 'dls/Toggle',
+  component: Toggle,
+};
+
+const Template = () => {
+  const [isChecked, setIsChecked] = useState(false);
+
+  return (
+    <Toggle
+      isChecked={isChecked}
+      onClick={() => {
+        setIsChecked(!isChecked);
```

Comment:
nit: Should be a function since useState is async e.g. setIsChecked((prevIsChecked) => !prevIsChecked)